documentation/src/docs/javadoc.md

```diff
@@ -40,13 +42,13 @@ The Batch Client allows:
 - generating a streamCut that is a bounded distance from another streamCut.
 
 ## Byte Client
-_[Byte Client](https://github.com/pravega/pravega/wiki/PDP-30-ByteStream-API)_ is a byte-oriented API providing a way 
+_[Byte Client](https://www.pravega.io/docs/latest/javadoc/clients/io/pravega/client/ByteStreamClientFactory.htmll)_ is a byte-oriented API providing a way 
```

Comment:
This link is broken, there is an extra `l` at the end of the link. Please, check all the links for issues like this.

Reply:
Removed.